### PR TITLE
Fix route names and POST logic per suggestion from PaulYuk

### PR DESCRIPTION
**Pull request by ggailey777**
## Purpose

* Added a route to bring the endpoint URLs in line with the other lang samples.
* Updated the POST method to also expect `age` 
* Updated test files

## Does this introduce a breaking change?

```
[X] Yes (POST body has changed)
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[X] Feature
[X] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Start the func.exe
* Run the POST using RESTClient.

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by ggailey777** [comment]
@paulyuk @bigmike @eamonoreilly

**Review by paulyuk — APPROVED** [review]
LGTM

File: src/test.http [hunk 1]
```diff
@@ -1,10 +1,11 @@
-GET http://localhost:7071/api/httpGetFunction HTTP/1.1
+GET http://localhost:7071/api/httpget?name=Awesome%20Developer HTTP/1.1
```
Comment (paulyuk) [inline src/test.http]:
```suggestion
GET http://localhost:7071/api/httpget?name=World HTTP/1.1
```

**Review by paulyuk — CHANGES_REQUESTED** [review]
LGTM, small suggestion

File: README.md [hunk 2]
```diff
@@ -13,63 +15,83 @@ urlFragment: functions-quickstart-powershell-azd
...
-git clone https://github.com/Azure-Samples/functions-quickstart-powershell-azd.git
-```
+## Initialize the local project
+
+You can initialize a project from this `azd` template in one of these ways:
+
++ Use this `azd init` command from an empty local (root) folder:
+
+    ```powershell
+    azd init --template functions-quickstart-javascript-azd
```
Comment (BigMorty) [inline README.md]:
```suggestion
    azd init --template functions-quickstart-powershell-azd
```

**Review by BigMorty — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ggailey777) [reply README.md]: Great catch on my `cut-and-paste` process 😉

**Pull request merged**